prov/efa/configure.m4

```diff
@@ -170,6 +172,21 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 
 	AM_CONDITIONAL([HAVE_EFADV_CQ_EX], [test $efadv_support_extended_cq -eq 1])
 
+	# IBV_QUERY_QP_DATA_IN_ORDER_ALIGNED_128_BYTES is the flag we need test,
+	# but it is flag. If it is not there, we should disable it.
+	AS_IF([test $efa_support_data_in_order_aligned_128_byte -eq 1],
```

Comment:
I'm not great with .m4 files, but it looks to me like the test here should be on a different variable name?  It looks to me like you will  never bother to grep the header because on entry the variable will be 0.

Reply:
The variable `efa_support_data_in_order_aligned_128_byte` was initialized to 1 earlier.